visualc_net/dosbox.vcxproj

```diff
@@ -69,7 +69,7 @@
       <CompileAs>Default</CompileAs>
     </ClCompile>
     <Link>
-      <AdditionalDependencies>opengl32.lib;sdl_net.lib;winmm.lib;libpng16d.lib;sdlmaind.lib;sdl.lib;odbc32.lib;odbccp32.lib;ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>opengl32.lib;sdl_net.lib;winmm.lib;libpng16d.lib;sdlmaind.lib;sdl.lib;odbc32.lib;odbccp32.lib;ws2_32.lib;opusfile.lib;opus.lib;ogg.lib;libspeexdsp.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
I suggest putting changes to VisualStudio files in a sperate commit - it will make it easier to submit the changes upstream.

Reply:
Separated into 6b6344b3.